repo2docker/buildpacks/conda/install-miniforge.bash

```diff
@@ -42,11 +41,6 @@ echo 'update_dependencies: false' >> ${CONDA_DIR}/.condarc
 # avoid future changes to default channel_priority behavior
 conda config --system --set channel_priority "flexible"
 
-# do all installation with mamba
-time conda install -y mamba==${MAMBA_VERSION}
```

Comment:
Maybe we want to keep `mamba install -y mamba==${MAMBA_VERSION}` to allow pinning versions more recent than what's in the mambaforge release? We always had this for conda before adopting mamba, because we could update conda to be more recent than a miniconda release.

Reply:
Why not, although it may defeat the purpose of saving this call to the solver.

Reply:
But at least it's mamba instead of conda, and should be quick if the version matches what is already isstalled, right?

Reply:
Ah yes I can use mamba!

Reply:
use:

```bash
mamba install -y -S mamba==${MAMBA_VERSION}
```

and it shouldn't download repodata or anything if the version is what's already installed

Reply:
Hm, nevermind. My env is weird and mamba does seem to fetch repodata even if specs are already satisfied. We could add checks beforehand, but I don't think there's much point since repodata will be fetched immediately on the next line anyway

Reply:
I have a reservation about this: what if the set mamba version causes a downgrade of conda? I think that conda downgrades are only allowed if a certain configuration parameter is set.

Reply:
@minrk let me know what you think!

Reply:
Realistically, we are never going to request mamba older than what's in mambaforge, so it should only cause upgrades, not downgrades. We always used this in the past to get a conda update that's not in miniconda yet, and would only use this if there's a mamba update (or conda even!) that's *not* in the latest mambaforge, but we still want.

This could be moot if you think mambaforge will be updated on ~every conda or mamba release. Miniconda was pretty far from that, though.

Reply:
OK, I re-added the update of mamba to the version specified.